System.Compiler/Reader.cs

```diff
@@ -1924,6 +1924,10 @@ private AssemblyNode.PostAssemblyLoadProcessor ReferringAssemblyPostLoad
     private TypeNodeList GetTypeParametersFor(int parentIndex, Member parent){
       GenericParamRow[] genericParameters = this.tables.GenericParamTable;
       TypeNodeList types = new TypeNodeList();
+      TypeNodeList savedTypes = this.currentTypeParameters;
+      this.currentTypeParameters = types;
+      try
+      {
       int i = 0, n = genericParameters.Length, j = n-1;

```

Comment:
Rebracer may be a nice solution for keeping formatting for the project the same for everybody, works well for me at work.